test/e2e/e2e.go

```diff
@@ -340,7 +340,7 @@ func logClusterImageSources() {
 // TODO: These should really just use the GCE API client library or at least use
 // better formatted output from the --format flag.
 
-// Returns master & node image string, or error
+// Returns control plane & data plane image string, or error
```

Comment:
`control plane & data plane node image` to be specific, wdyt?

Reply:
meant to fix this when going through, thanks for pointing it out. will push up these changes shortly.